docs/developer-docs/integrations/sns/managing/sns-asset-canister.md

```diff
@@ -23,11 +23,14 @@ An asset canister must first be deployed before control of it can be handed over
 
 The general overview of deploying an asset canister during an SNS launch is as follows:
 - First, the asset canister must be created with or upgraded to a Wasm file from [dfx 0.14.1+](https://github.com/dfinity/sdk/blob/release-0.14.1/src/distributed/assetstorage.wasm.gz).
-- Then, the dapp should hand control of the asset canister over to the SNS by setting the following permissions:
-    - The SNS governance canister is given `Commit` permissions. This is done by the previous developer using the `grant_permissions` command (see the granting permissions section below), otherwise the SNS must grant this permission once it is a controller.
-    - A whitelist of principals with `Prepare` permissions is created to give certain individuals the permission to upload changes to the asset canister. Changes must be approved through a proposal before they are applied to the asset canister. 
-    - The user or developer creating the SNS should remove their own personal permissions. 
-- Lastly, the SNS's function should be registered to commit the configuration.
+- Then, the dapp developers hand over control of the entire dapp to the NNS, including the asset canister.
+- A single NNS proposal creates the dapp's SNS canisters, initializes them, and hands the dapp canisters over to the SNS. The decentralization swap begins.
+- The SNS community then upgrades the dapp canister through an SNS proposal and sets the privileges in the dapp. 
```

Comment:
What dapp canister is this talking about?

Reply:
Added clarification